### PR TITLE
I5 setup storybook js

### PR DESCRIPTION
**Pull request by ThePhisch**
Installed storybook and ported the first story components. Installed further dependencies including `@primer/css`, `bulma`, `sass`, `sass-loader` as needed.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```
Comment (maany) [inline .npmrc:1]: what package requires the `legacy-peer-deps` flag to be enabled?

**Review by maany — CHANGES_REQUESTED** [review]
Looks good! If we can avoid using `legacy-peer-deps` it would be best. But for some packages, it is unavoidable.
Also, can we move the stories inside the `src` directory? and rename it to `component-library`?

File: .npmrc (hunk 1 above)
Comment (ThePhisch) [reply .npmrc:1]: I was prompted to add this flag during the installation of `storybook`. It seems to be a [known problem](https://github.com/storybookjs/storybook/issues/18298).

**Comment by ThePhisch** [comment]
> Also, can we move the stories inside the `src` directory? and rename it to `component-library`?

I will rename `src/stories` to `src/component-library`, as discussed.

**Review by maany — CHANGES_REQUESTED** [review]
Can we already migrate to Storybook 7
https://chromatic-ui.notion.site/Storybook-7-migration-guide-dbf41fa347304eb2a5e9c69b34503937 

Also, can you refactor the component-library directory to have 2 sub-directories, one for components and one for sass/scss

The components can import from the sass/scss

**Comment by maany** [comment]
can you rebase?

**Pull request merged**